README.md

```diff
@@ -36,6 +36,9 @@ helmversion = "1.3"
 branchname = "release-X.Y.Z"
 ```
 
+As last step need to bump the [_index.md](https://github.com/goharbor/website/blob/master/docs/_index.md) file with the new version and fix the links in the file
```

Comment:
Could be "As the last step, update the ... file with the new version number and correct links."

Reply:
done!